src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/ClusterMonitorActor.scala

```diff
@@ -235,11 +233,6 @@ class ClusterMonitorActor(val cluster: Cluster,
         case _ => Future.successful(NotReadyCluster(googleStatus))
       }
     } yield result
-
-    // Recover from Google 404 errors, and assume the cluster is deleted
```

Comment:
why remove this?

Reply:
1. because the DAO is no longer throwing `CallToGoogleApiFailedException`
2. `getClusterStatus` actually handles this and returns "Deleted"